doc/level1.rst

```diff
@@ -710,6 +701,79 @@ way:
 4. Set ``peers[gossip-addr].gossip_key`` to the value of the
    ``keydata`` attribute.
 
+E-Mail Drafts
+-------------
+
+During message composition, the user may want to close the dialog and
+resume composition at a later point.  To preserve the session state,
+messages are typically saved as "Drafts" in a special folder.
+Messages from this folder may also be resumed from a different MUA.
+
+To prevent the cleartext of a message draft from leaking to the
+provider, the MUA can encrypt the draft message to itself (but not the
+recipients) before uploading.  It MUST encrypt in this way if the
+message is going to be sent encrypted, and SHOULD encrypt if the user
+might decide to encrypt at a later point, or otherwise keep the draft
+local only.  Drafts encrypted in this way SHOULD be stored in PGP/MIME
+format.
+
+To allow storage of drafts independently from the user's secret key,
+encrypted drafts do not need to be signed.  Conversely, a MUA SHOULD
+ignore the signature status of encrypted drafts.
+
+Storing Draft State
++++++++++++++++++++
+
+To store information about whether a message should be encrypted when
+sent, an `Autocrypt-Draft-State` header MAY be added to the draft when
+it is stored.
+
+The `Autocrypt-Draft-State` header consists of a list of attributes
+with the same syntax as the :ref:`Autocrypt header<autocrypt-header>`
+itself, which also supports critical and non-critical attributes
+following the same semantics.  There are three defined attributes,
+all of which have a binary value of `yes` or `no`:
+
+* The critical `encrypt` attribute specifies whether the message would
+  be sent encrypted or not, at the time of saving the draft. It MUST
+  be present in the header.
+* The non-critical `_is-reply` attribute indicates whether the message
+  is composed as a reply to an encrypted message. This affects the
+  :ref:`Autocrypt recommendation<final-recommendation-phase>`. It is
+  optional and defaults to `no`.
+* The non-critical `_by-choice` attribute indicates whether the value
+  of the `encrypt` attribute was made by immediate user choice. It is
+  optional and defaults to `no`.
+
+Example::
+
+    Autocrypt-Draft-State: encrypt=yes; _by-choice=yes;
+
+.. note::
+
+    It would be desirable to store this header in the encrypted
+    payload.  This is missing here because of technical limitations in
+    some MUAs.
+
+Autocrypt Gossip headers for drafts
++++++++++++++++++++++++++++++++++++
+
+When opening a draft that indicates it should be encrypted when sent
+via the ``Autocrypt-Draft-State`` header, the resuming MUA might be
+lacking keys for recipients that were originally available when the
+draft was stored.
+
+To solve this problem, a MUA MAY make use of the `Autocrypt
+Gossip<autocryt-gossip>` mechanism to include the keys of intended
```

Comment:
link is misspelled.  can just be written as:

    :ref:`key-gossip`

no need to include link text explicitly, it'll get picked up from the markup.